lib/monitoring/aws-elasticache/ElastiCacheClusterMonitoring.ts

```
@@ -157,6 +172,20 @@ export class ElastiCacheClusterMonitoring extends Monitoring {
       this.cpuUsageAnnotations.push(createdAlarm.annotation);
       this.addAlarm(createdAlarm);
     }
+
+    for (const disambiguator in props.addRedisEngineCpuUsageAlarm) {
```

Should throw an error if this is attempted when it's not a Redis cluster.

---

done (see above)